docs/getting-started/for-dapp-devs/send_transactions_on_axon_via_metaMask.md

```diff
@@ -16,14 +16,16 @@ This guide provides instructions for sending transactions on Axon via MetaMask a
 
 ### 1.1 Local Setup
 
-[Download Axon](https://github.com/axonweb3/axon), find <b>Install</b> in <b>README</b>, and run the following commands to start the node:
+[Download Axon](https://github.com/axonweb3/axon), find <b>Compile</b> in <b>README</b>, and run the following commands to start the node:
 
-`cargo run --release -- -c devtools/chain/config.toml -g devtools/chain/genesis_single_node.json`
+```bash
+target/release/axon run --config devtools/chain/config.toml
```

Comment:
This command is not enough to start a Axon chain, missing `axon init`.

Reply:
suggestion:
reuse the quick_start doc:

https://github.com/sunchengzhu/axon-docs/blob/313d023c11cfdfec12d6c03483c31aee70e01c49/docs/getting-started/for-dapp-devs/deploy_solidity.md?plain=1#L14

Reply:
> suggestion: reuse the quick_start doc:
> 
> https://github.com/sunchengzhu/axon-docs/blob/313d023c11cfdfec12d6c03483c31aee70e01c49/docs/getting-started/for-dapp-devs/deploy_solidity.md?plain=1#L14

Modified in https://github.com/axonweb3/axon-docs/pull/205/commits/be24182d8cacf543bec5a5199a62bb12a9b54624